src/components/10-atoms/radio/react/DemoRadioButtonReact.jsx

```diff
@@ -6,6 +6,7 @@ const DemoRadiobuttonsControlled = () => {
   const [button, setButton] = useState(true);
```

Comment:
I don't really understand what more this demo-file is actually demoing now. Comments would be helpful.

Reply:
// rendering axa-radios invisible cleanly removes them from DOM without exceptions being thrown

is the comment describing the 'what more'. But that one is in ui.test.js, so you're right I should add something also in DemoRadioButtonReact.jsx.

Reply:
Done